stylesheets/stylesheet.css

```diff
@@ -40,5 +33,11 @@ font-size: 2rem;
     border: 1px solid #5BC0BE;
     padding: 10px;
     border-radius: 10px;
-    margin-bottom: 20px;
+   margin: 20px 20px;
```

Comment:
if it's {margin: 20px 20px}, you can just write once->  {margin: 20x;}

Reply:
good point! resolved